source/fundamentals/crud/read-operations/specify-query.txt

```diff
@@ -0,0 +1,331 @@
+   { "_id": 4, "make": "Kiesel", "models": ["Ares", "Vader", "Solo"], "establishedYear": 2015 }
+   { "_id": 5, "make": "Ibanez", "models": ["RG", "AZ"], "establishedYear": 1957, "rating": 7 }
+   { "_id": 6, "make": "Strandberg", "models": ["Boden", "Salen"], "establishedYear": 1982 }
+
+The following ``Guitar`` class models the documents in this collection. The 
+{+driver-short+} uses this class to automatically map these documents to and 
+from BSON.
+
+.. TODO: Add link to Automapping page.
+
+.. literalinclude:: /includes/fundamentals/code-examples/specify-query/Guitar.cs
+   :language: csharp
+   :copyable:
+   :dedent:
+
+Literal Values
+--------------
+
+Literal value queries return documents with an exact match to your query filter.
+
+The following example specifies a query filter as a parameter to the ``Find()`` 
+method, using the ``Guitar`` class. The query matches all documents where the
+``make`` field equals "Fender".
+
+.. io-code-block::
+   :copyable:
+
+   .. input:: /includes/fundamentals/code-examples/specify-query/FindEqPOCO.cs
+      :language:  csharp
+
+   .. output:: 
+      :language: json
+      :visible:
+
+      { "_id" : 1, "make" : "Fender", "models" : ["Stratocaster", "Telecaster"], "establishedYear" : 1946, "rating" : 9 }
+
+The following example creates a query filter using builders that matches all 
+documents where the ``make`` field equals "Fender".
+
+.. io-code-block::
+   :copyable:
+
+   .. input:: /includes/fundamentals/code-examples/specify-query/FindEqBuilder.cs
+      :language:  csharp
+
+   .. output:: 
+      :language: json
+      :visible:
+
+      { "_id" : 1, "make" : "Fender", "models" : ["Stratocaster", "Telecaster"], "establishedYear" : 1946, "rating" : 9 }
+
+.. tip:: Find All Documents
+
+   Using an empty query filter matches all documents in the collection. Create
+   an empty query filter with builders as follows:
+
+   .. code-block:: csharp
+
+      var result = _guitarsCollection.Find(Builders<Guitar>.Filter.Empty).ToList();
+
+To learn more about using builders, see :ref:`csharp-builders`.
+
+Comparison Operators
+--------------------
+
+Comparison operators analyze the value in a document against the specified value
+in your query filter. Common comparison operators include:
+
+.. list-table::
+   :widths: 30 30 40
+   :header-rows: 1
+
+   * - Expression
+     - Builder
+     - Description
+
+   * - ``>``
+     - ``Gt()``
+     - Greater than
+
+   * - ``<=``
+     - ``Lt()``
+     - Less than or equal to
+
+   * - ``!=``
+     - ``Ne()``
+     - Not equal to
+
+For a full list of comparison operators, see the :manual:`Comparison
+Query Operators </reference/operator/query-comparison/>` page.
+
+The following example specifies a query filter as a parameter to the ``Find()``
+method, using the ``Guitar`` class. The query matches all documents where the
+``establishedYear`` field is greater than ``1985``.
```

Comment:
I removed "using the ``Guitar`` class from these paragraphs since we now state at the top that the ``Guitar``class is used in the examples for automapping. I left the rest of the paragraph the same as I think it's important to still have the focus of the example be specifying the query since that's the topic of the page, rather than the result of the query.